app/views/events/index.html.erb

```diff
@@ -27,6 +27,16 @@
         <td>
           <%= link_to t('.show', :default => t("helpers.links.show")),
                       event_path(event), :class => 'btn btn-default btn-xs' %>
+          <% if (user_signed_in? && event.teamsport==false)%>
+              <% if event.users.where("user_id = ?", current_user.id).exists? %>
+                  <%= link_to 'Leave Event', join_event_path(event), :method => "put",
+                              :class => 'btn btn-default', id: :leavebutton%>
+                  Participating
```

Comment:
Is localization now an impicit acceptance criteria or not? If yes we should maybe add localized button labels.

Reply:
technically it is...